R/utils_get_db.R

```diff
@@ -217,7 +217,7 @@ get_pkg_info <- function(pkg_name, db_name = golem::get_golem_options('assessmen
 #' @noRd
 get_metric_weights <- function(db_name = golem::get_golem_options('assessment_db_name')){
   dbSelect(
-    "SELECT name, weight
+    "SELECT name, weight, is_perc
```

Comment:
@Robert-Krajcik This is why you are getting NA's. You also need to return the `id` from the metric table.